### PR TITLE
Feature/deployment ready

### PR DESCRIPTION
**Pull request by abel-asfaw**
Fully prepared for deployment

### CODE REVIEW

File: Covid Lunches/express/express-bootstrap/views/inner-page.njk [hunk 1]
```diff
@@ -114,7 +114,9 @@
         <h2 id="LocationLabel" style="color:forestgreen"> Old Location</h2>
         </div>
 
-        <div id="googleMap" style="margin: 0px 0px 0px 0px; width:100%;height:400px"></div>
+        <div id="googleMap" style="margin: 0px 0px 0px 0px; width:100%;height:400px">
+            <div id="key" style="display: none;"> {{ mapKey }} </div>
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/views/inner-page.njk:118]: I get it!

Comment (F-421) [reply Covid Lunches/express/express-bootstrap/views/inner-page.njk:118]: Also hidden on heroku

File: Covid Lunches/express/express-bootstrap/public/js/mapscript.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 function callMapAPI() {
 
     var script = document.createElement('script');
 
     /*safely access the key*/
-    var my_key = config.MAP_KEY;
-    script.src = 'https://maps.googleapis.com/maps/api/js?key=' + my_key + '&callback=myMap';
+    var my_key = document.getElementById("key").innerHTML;
+    script.src = `https://maps.googleapis.com/maps/api/js?key=${my_key}&callback=myMap`;
     script.async = true;
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/mapscript.js:9]: Cool. Thanks!

**Review by F-421 — APPROVED** [review]
Looks good. Approved.

**Comment by abelasfaw1** [comment]
wtf

**Review by rucbarr — APPROVED** [review]
Thumbs Up. (For the final time.) (Goodbye 447.) (You were loved.)

**Pull request merged**